code/parachain/frame/cosmwasm/src/runtimes/abstraction.rs

```diff
@@ -126,7 +126,7 @@ impl Gas {
 	}
 
 	pub fn pop(&mut self) {
-		let child = self.checkpoints.pop().unwrap();
+		let child = self.checkpoints.pop().expect("non empty");
```

Comment:
That’s just noise IMO.

Reply:
we have linter, now contracts are part for common CI. so i fixed linter.  other option fix by supperess (ugly) or disable all unwrap?

Reply:
I’d much rather we got rid of the lint check.

Reply:
btw, `gas_checkpoint_pop` allows updates and bugs to CW brick runtime easy :) it could just return error empty checkpoint.

Reply:
bad code

Reply:
Returning error works for me.  I actually took a cursory look at whether we could have some kind of scoping here so rather than push and pop operations have push which returns a guard object which pops as RAII.  Didn’t see an obvious way to do it though.

Reply:
https://github.com/ComposableFi/composable/pull/3815

Reply:
real fix in CW repo side i guess.  so it it is like big work.